nml/actions/action0properties.py

```diff
@@ -291,6 +295,26 @@ def ctt_list(prop_num, *values):
             raise generic.ScriptError("Value of cargolist property must be an array", value.pos)
     return [VariableByteListProp(prop_num, [[ctype.reduce_constant().value for ctype in single_item_array.values] for single_item_array in values])]
 
+def VariableWordListProp(num_prop, data):
+    return VariableListProp(num_prop, data, 2)
+
+def accepted_cargos(prop_num, *values):
+    # values may have multiple entries, if more than one item ID is set (e.g. multitile houses)
+    # Each value is an expression.Array of cargo types and amount arrays
+    cargos = []
+    for value in values:
+        if not isinstance(value, Array) or len(value.values) > 16:
+            raise generic.ScriptError("accepted_cargos must be an array with no more than 16 values", value.pos)
+        tile_cargos = []
+        for cargo_amount_pair in value.values:
+            if not isinstance(cargo_amount_pair, Array) or len(cargo_amount_pair.values) != 2:
+                raise generic.ScriptError("Each element of accepted_cargos must be an array with two elements: cargoid and amount", cargo_amount_pair.pos)
+            cargo_id = cargo_amount_pair.values[0].reduce_constant().value
+            cargo_amount = cargo_amount_pair.values[1].reduce_constant().value
```

Comment:
How about a 
```
 if isinstance(cargo_amount, ConstantNumeric):
        generic.check_range(cargo_amount.value, 0, 8, "cargo acceptance", cargo_amount.pos)
```
cargo_id could also be checked for range, but probably does not help much.

Reply:
for industry tiles it can be <0 to counter `INDTILE_FLAG_ACCEPT_ALL`  effect

Reply:
good point, I forgot that :)